### ISSUE TITLE
Upgrade RN to 0.61 from 0.59

### ISSUE THREAD

**Issue by almasaeed2010** [statonlab/Treesnap-mobile#627]
We are running behind on RN updates. This is a major upgrade so we'll need to go through every feature and test it

**Comment by almasaeed2010** [comment]
Builds on android but doesn't on iOS

**Comment by almasaeed2010** [comment]
It's still not compiling:

```
/node_modules/react-native/ReactAndroid/src/main/jni/first-party/fb/include/fb/assert.h:13:1: Unknown type name 'namespace'
/Users/Almsaeed/Work/TreeSnap/ios/TreeSnap/main.m:8:9: While building module 'UIKit' imported from /Users/Almsaeed/Work/TreeSnap/ios/TreeSnap/main.m:8:
```

**Comment by almasaeed2010** [comment]
I published a downgraded version (RN 0.59) but we'll need to update soon since Apple will no longer accept submissions that use the old WebViews implementation by December 2020

**Comment by almasaeed2010** [comment]
done
